lib/project_types/theme/messages/messages.rb

```diff
@@ -2,12 +2,38 @@
 module Theme
   module Messages
     MESSAGES = {
-      ensure_themekit_installed: {
-        downloading: "Downloading Themekit %s",
-        failed: "Download failed",
-        successful: "Themekit installed successfully",
-        unsuccessful: "Unable to verify download digest",
-        verifying: "Verifying download...",
+      theme: {
+        create: {
+          creating_theme: "Creating theme %s",
+          checking_themekit: "Verifying Themekit",
```

Comment:
Should Theme Kit be two words to match https://shopify.github.io/themekit/ ?

(I realise it's already written as Themekit in this file, but since we're adding more occurrences maybe this the time to fix it).

cc @gfscott to confirm.

Reply:
Can confirm! It should always appear as **Theme Kit**.